imports/client/components/NotificationCenter.tsx

```diff
@@ -320,6 +322,32 @@ function ProfileMissingMessage(props: ProfileMissingMessageProps) {
   );
 }
 
+interface ChatNotificationMessageProps {
+  // TODO: add the requisite fields
+  cn: NotificationCenterChatNotification;
+  onDismiss: () => void;
+}
+function ChatNotificationMessage(props: ChatNotificationMessageProps) {
+  return (
+    <li>
+      <MessengerSpinner />
+      <MessengerContent dismissable>
+        <Link to={`/hunts/${props.cn.hunt._id}/puzzles/${props.cn.puzzle._id}`}>
```

Comment:
FTR, this could just be `/hunts/${props.cn.puzzle.hunt}/puzzles/${props.cn.puzzle._id}` and we wouldn't actually need the hunt sub at all, right?

(I think that's not true of pending guesses, since we need the hunt's config for URL transformation)

Reply:
I think you're right.  I'm just slightly paranoid about ever making a foreign key available if the target isn't already present client-side, since 1) we seem to be pretty aggressive about assuming that we can `findOne()` them, and 2) many, many of our bugs resulting in totally blank layout in old hunts were the result of attempting to traverse `undefined`s. 😛